portfolio/src/main/webapp/script.js

```diff
@@ -228,10 +228,63 @@ async function deleteComments() {
   displayServletContent(0);
 }
 
-/* Creates a map and adds it to the page */
+const cornellCoords = { lat: 42.449, lng: -76.474 };
```

Comment:
I recommend to capitalize all const vars like how you did in java.

Reply:
Done